konfera/models/event.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-
+from konfera.models import Location
```

Comment:
There is some problem with import (desnt seems to be working), please remove this import as it breaks the automated tests: https://travis-ci.org/pyconsk/django-konfera/builds/160476772 

If you do `location = models.ForeignKey('Location')` (notice that 'Location' is string), Django will handle it and you dont need import.